<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QMainWindow" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1582</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgb(228, 205, 255)}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Calendar" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>361</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>781</width>
      <height>361</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>781</width>
      <height>361</height>
     </size>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCalendarWidget QWidget{
background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(0, 0, 0);}
</string>
    </property>
    <property name="text">
     <string>Average Cycle Length</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chartButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:rgb(0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>showCycle</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cycleLengthSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {color:rgb(0, 0, 0);}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:rgb(0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>Log Period</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:rgb(0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>Remove Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statsButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:rgb(0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>showStatistics</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1582</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Calendar</class>
   <extends>QCalendarWidget</extends>
   <header>calendar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
